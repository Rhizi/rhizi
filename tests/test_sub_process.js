var spawn = require('child_process').spawn;

var ls = spawn('ls');
ls.stdout.on('data', function (data) {
    console.log('stdout ' + data.length);
});
ls.on('close', function(code, signal) {
    console.log('closed, ' + code + ', ' + signal);
});
