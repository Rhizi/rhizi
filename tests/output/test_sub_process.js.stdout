stdout 200
closed, 0, null
